move manifests

Gkrumbach07 opened the pull request:
This PR overwrites manifests here with manifests from [osc manifests](https://github.com/operate-first/apps/tree/master/odh-manifests/osc). There is no merging taking place yet.

**apps changed**
- [ ] jupyterhub
- [ ] odhseldon
- [ ] superset
- [ ] trino

https://github.com/open-services-group/devsecops/issues/8

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **anishasthana** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @anishasthana` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2Fodh-manifests).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/operate-first/odh-manifests/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["anishasthana"]} -->

Gkrumbach07 commented:
**Jupyterhub**
- removed `jupyterhub-ha-sidecar` config not present in osc
- added node_tolerations and node_affinity to profiles
- set groups / admin groups

HumairAK commented:
@Gkrumbach07 is this ready for review, or are you still working on this? if it's still in progress can you add a "WIP" prefix to the PR title?

Gkrumbach07 commented:
> @Gkrumbach07 is this ready for review, or are you still working on this? if it's still in progress can you add a "WIP" prefix to the PR title?

@HumairAK Everything is moved over besides the trino-dev manifests, which doesn't have a spot in this repo. I wasn’t sure if we were just overwriting the manifests or merging them. If overwriting then this is ready to be reviewed, if we are merging them then its WIP.
